backend/src/nodes/utils/ncnn_model.py

```diff
@@ -5,10 +5,16 @@
 from typing import Dict, List, Tuple, Union
 
 import numpy as np
-import onnx.numpy_helper as onph
-from onnx import TensorProto
 from sanic.log import logger
 
+# Don't want not having onnx to crash this
+try:
+    import onnx.numpy_helper as onph
+    from onnx import TensorProto
+except:
+    TensorProto = {}
+    onph = {}
```

Comment:
I think I did this because it got very mad at me for setting them to none

Reply:
Try running with `npm start` and onnx uninstalled. It won't import the ncnn nodes at all. It's kind of funny because you left a comment saying "Don't want not having onnx to crash this," but it makes not having onnx break ncnn. :smile: 

![image](https://user-images.githubusercontent.com/55669985/187303321-4c41e78a-1401-4d69-a470-4b01b5dc8d3c.png)

Reply:
Oops lol